.travis.yml

```diff
@@ -2,19 +2,24 @@ esudo: false
 branches:
   only:
   - master
+  - develop
 language: node_js
 node_js:
 - '6'
 before_install:
 - 'pwd && ls -als'
+- curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 0.23.2
+- export PATH="$HOME/.yarn/bin:$PATH"
```

Comment:
Thanks for the "heads-up", I did not realise TravisCI provided such an old default Yarn version `0.17` (when latest is `0.27.5`).
I am trying this in `r2-streamer-js` (note: no explicit version, should fetch latest):
https://github.com/edrlab/r2-streamer-js/commit/5ff0b49ee5db7648f8e52175e2ce75a15e7cbed6
...let's see:
https://travis-ci.org/edrlab/r2-streamer-js/builds/257732983